website/source/docs/upgrade-specific.html.md

```diff
@@ -48,18 +48,22 @@ in OSS or `web.service.ns1.dc1.consul` for Enterprise.
 
 Consul has changed the semantics of query counts in its [telemetry](/docs/agent/telemetry.html#metrics-reference).
 `consul.rpc.query` now only increments on the *start* of a query (blocking or non-blocking), whereas before it would
-measure when blocking queries polled for more data. The gauge `consul.rpc.queries_blocking` has been added for a more
+measure when blocking queries polled for more data. The `consul.rpc.queries_blocking` gauge has been added
 to more precisely capture the view of *active* blocking queries.
 
-### Vault: default for `http_max_conns_per_client` to low to run Vault properly
+### Vault: default `http_max_conns_per_client` too low to run Vault properly
 
-Consul 1.7.0 introduced limiting of connections per client ((docs)[https://www.consul.io/docs/agent/options.html#http_max_conns_per_client]). The default is 100, but Vault allows up to 128 which causes problems. If you want to use Vault with Consul 1.7.0, you should change it to 200. Starting with Consul 1.7.1 this is the new default.
+Consul 1.7.0 introduced [limiting of connections per client](/docs/agent/options.html#http_max_conns_per_client). The default value
+was 100, but Vault used up to 128 which caused problems. If you want to use Vault with Consul 1.7.0, you should change the value to 200.
```

Comment:
I changed "allows" to "used". Is that accurate? My understanding is that vault would initiate the connection, so "used" seems like a more correct term in this case.

Reply:
Vault has a setting I believe to limit how many concurrent connections it will make. That value defaults to 128 and whether its user-overridable I am not sure.

Reply:
Ok, makes sense. I'll change this to "may use up to"